package.json

```diff
@@ -88,7 +88,7 @@
     "browserify": "14.5.0",
     "cross-env": "3.2.4",
     "electron": "1.8.4",
-    "electron-builder": "13.11.1",
+    "electron-builder": "20.10.0",
```

Comment:
Need to ensure that we don't run into any regressions with this change.

Reply:
Already tested, no regressions. Created builds on Windows.